static/app/components/replays/replayRageClickSdkVersionBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from '@emotion/styled';
+
+import replaysDeadRageBackground from 'sentry-images/spot/replay-dead-rage-changelog.svg';
+
+import {Button} from 'sentry/components/button';
+import PageBanner from 'sentry/components/replays/pageBanner';
+import {IconBroadcast} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {MIN_DEAD_RAGE_CLICK_SDK} from 'sentry/utils/replays/sdkVersions';
+import useDismissAlert from 'sentry/utils/useDismissAlert';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export default function ReplayRageClickSdkVersionBanner() {
+  const organization = useOrganization();
+
+  const {dismiss, isDismissed} = useDismissAlert({
+    key: `${organization.id}:replay-rage-dead-click-sdk-version-banner-v1`,
+  });
+
+  if (isDismissed) {
+    return null;
+  }
+
+  return (
+    <PageBanner
+      button={<Button priority="primary">Read Changelog</Button>}
```

Comment:
```suggestion
      button={<Button priority="primary">{t('Read Changelog')}</Button>}
```

Reply:
![7z0dc9](https://github.com/getsentry/sentry/assets/187460/fb3ef33f-b536-4500-8820-1acf56585ed7)

oops, forgot about the `onClick` part

Reply:
Ah thought you were just dogfooding dead clicks ;)